pkg/operator/sync_openshiftcontrollermanager_v311_00_test.go

```diff
@@ -119,28 +119,75 @@ func TestConfigMapControllerDisabling(t *testing.T) {
 		result              map[string][]string
 	}{
 		{
-			name: "ControllersEnabled",
+			name: "CapabilitiesEnabled",
 			knownCapabilities: []v1.ClusterVersionCapability{
 				configv1.ClusterVersionCapabilityBuild,
 				configv1.ClusterVersionCapabilityDeploymentConfig,
+				configv1.ClusterVersionCapabilityImageRegistry,
 			},
 			enabledCapabilities: []v1.ClusterVersionCapability{
 				configv1.ClusterVersionCapabilityBuild,
 				configv1.ClusterVersionCapabilityDeploymentConfig,
+				configv1.ClusterVersionCapabilityImageRegistry,
 			},
 			result: map[string][]string{"controllers": {"*"}},
 		},
 		{
-			name: "ControllersDisabled",
+			name: "BuildCapDisabled",
+			knownCapabilities: []v1.ClusterVersionCapability{
+				configv1.ClusterVersionCapabilityBuild,
+			},
+			enabledCapabilities: []v1.ClusterVersionCapability{},
+			result: map[string][]string{
+				"controllers": {"*",
+					"-openshift.io/build",
+					"-openshift.io/build-config-change",
+					"-openshift.io/builder-rolebindings",
```

Comment:
https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/master/pkg/operator/configobservation/controllers/capability_builds.go#L24-L27 for the full list. `openshift.io/builder-serviceaccount` is missing.